common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -213,8 +214,10 @@ public static List<Component> wrapTooltips(List<Component> tooltips, int maxWidt
     }
 
     public static List<Component> splitComponent(Component component, int maxWidth) {
-        return McUtils.mc().font.getSplitter().splitLines(component, maxWidth, Style.EMPTY).stream()
```

Comment:
i assume the issue was just that this function returns an empty list if given a blank/newline component?

Reply:
Yes, sorry for not specifying.